hack/make/manifests/kubernetes.mk

```diff
@@ -5,8 +5,8 @@ define generate_k8s_manifest
 		--set installCRD=true \
 		--set platform="kubernetes" \
 		--set manifests=true \
-		--set olm="${OLM}" \
-		--set image="$(IMAGE_URI)" > $(2)
+		--set olm=${OLM} \
```

Comment:
`${}` and `$()` are equivalent to my knowledge, so let's use only one form to be more consistent

Reply:
I think @avorima is right so please use one style.